### DISCUSSION TITLE
Activity on profile

### DISCUSSION THREAD

**Discussion by abdul-36** [community/community#64097] [Actions]
### Select Topic Area

Show & Tell

### Body

Hello everyone i have an issue with my activity shown on my profile, my commits activity is not show on profile can anyone provide me any solution 

![image](https://github.com/community/community/assets/94354625/4c7bd97e-b4f9-4daf-a1f3-a8a7bd3bd297)

**Comment by christophercouch** [comment]
Hi @abdul-36,

There are several reasons why commits might not show up on your GitHub profile activity:

Private Contributions: If you're committing to private repositories and haven't opted to show private contributions on your profile, they won't appear. To display private contributions, go to your profile and click on Edit profile. Under Contribution settings, ensure that Private contributions is checked.

Email Address: Make sure the email address you use for your commits is associated with your GitHub account. If it's not, the commits won't be linked to your profile. You can check the email used in your commit by navigating to the commit in question and seeing what email was used. If this email is not linked to your GitHub account, you'll need to add it. Go to Settings > Emails and add the necessary email address.

Contribution Date: GitHub only counts contributions from the past year. So, if you're looking at older commits, they might not show up on your activity graph.

Repository Forks: Commits made in forks will not count towards your contributions unless a Pull Request is opened. If you want these commits to appear on your profile, you might consider creating a Pull Request or pushing directly to the main repository (if you have permissions).

Merged PRs: Only merged PRs, reviewed PRs, and issues opened by you will appear on your contribution graph. If you have opened a PR and it hasn't been merged, or if you've pushed to a branch but haven't opened a PR, it might not appear.

Pushed Commits: If you've made commits but haven't pushed them to GitHub, they won't show up. Ensure you've pushed all your changes to the remote repository.

Check GitHub Status: Sometimes there might be delays or issues with GitHub itself. You can check if GitHub is experiencing any issues by visiting their status page: [GitHub Status](https://www.githubstatus.com/).

If you've checked all of these and are still experiencing issues, it might be worth reaching out to GitHub support directly.

Hope this helps!
